src/app/TestStatusTable.tsx

```diff
@@ -72,7 +91,7 @@ export const TestStatusTable: React.FunctionComponent<TestStatusTableProps> = ({
   const columns = ["Name", "Status", "Synced with upstream?"];
 
   return (
-    <PageSection isWidthLimited>
+    <PageSection hasBodyWrapper isWidthLimited>
```

Comment:
another note that `hasBodyWrapper` should be true by default https://staging.patternfly.org/components/page#pagesection